File: src/ACMP754.java
import java.io.PrintWriter;
import java.util.Scanner;

public class ACMP754 {
    // your solution here
    private void solve(Scanner in, PrintWriter out) {
        int max = 0;
        for (int i = 0; i < 3; i++) {
            int mass = in.nextInt();
            if (mass >= 94 && mass <= 727) {
                max = Math.max(mass, max);
            } else {
                out.println("Error");
                return;
            }
        }

        out.println(max);
    }

    void run() {
        try (
            Scanner in = new Scanner(System.in);
            PrintWriter out = new PrintWriter(System.out)
        ) {
            solve(in, out);
        }
    }

    public static void main(String[] args) {
        new ACMP754().run();
    }
}

File: cpp/ACMP754.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main() {
    int maximum = INT32_MIN, tmp;
    for (int i = 0; i < 3; i++) {
        cin >> tmp;
        if (tmp >= 94 && tmp <= 727) {
            maximum = max(maximum, tmp);
        } else {
            cout << "Error" << endl;
            return 0;
        }
    }

    cout << maximum << endl;

    return 0;
}
